### PR TITLE
deployment tutorial

### PR DESCRIPTION
**Pull request by edublancas**
Added a tutorial to show how to deploy dashboards from the jupyterlab extension

----
:books: Documentation preview :books:: https://ploomber-doc--8.org.readthedocs.build/en/8/

### CODE REVIEW

File: doc/dashboards/jupyterlab-plugin.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+![](../static/dashboards/jupyterlab-plugin/5-confirm.png)
+
+Finally, you'll get a URL where you can track the deployment progress:
+
+
+![](../static/dashboards/jupyterlab-plugin/6-url.png)
+
+---
+
+Note: this guide was tested with `jupysql-plugin==0.1.6`
```
Comment (neelasha23) [inline doc/dashboards/jupyterlab-plugin.md:65]: This can be added to the first note

File: doc/dashboards/jupyterlab-plugin.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Once you create an account, go to [your account](https://www.platform.ploomber.io/account) to retrieve your API Key:
+
+![](../static/dashboards/jupyterlab-plugin/4-api-key.png)
+
+
+Copy it (ensure you copy it entirely!) and paste it in JupyterLab, then click on `Confirm`. JupyterLab will ask for confirmation to deploy, click on `Confirm`:
+
+![](../static/dashboards/jupyterlab-plugin/5-confirm.png)
+
+Finally, you'll get a URL where you can track the deployment progress:
```
Comment (neelasha23) [inline doc/dashboards/jupyterlab-plugin.md:58]: Can we add example of a URL here?

File: doc/dashboards/jupyterlab-plugin.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# JupyterLab plugin
+
+```{important}
+The plugin **is not** compatible with JupyterLab `4.x`, ensure you're running `3.x`: `pip install "jupyterlab<4"`
+```
+
+You can also use the JupyterLab plugin to deploy dashboards from JupyterLab.
```
Comment (neelasha23) [inline doc/dashboards/jupyterlab-plugin.md:7]: We can link the [doc](https://github.com/ploomber/jupysql-plugin/blob/main/doc/README.md) here

File: doc/dashboards/jupyterlab-plugin.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Installation
+
+```sh
+pip install jupysql-plugin
+```
+
+Once installed, a `Deploy Notebook` button will show up in the top bar:
```
Comment (neelasha23) [inline doc/dashboards/jupyterlab-plugin.md:15]: Does this come up automatically or we need to restart?

File: doc/dashboards/jupyterlab-plugin.md (hunk 3 above)
Comment (edublancas) [reply doc/dashboards/jupyterlab-plugin.md:7]: the link that you posted is developer documentation, this is meant for end-users

**Review by neelasha23 — APPROVED** [review]
looks good!

**Pull request merged**